index.html

```diff
@@ -1,2 +1,12 @@
-<html>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Strona</title>
+</head>
+<body>
+    <p>Moja strons</p>
```

Comment:
tutaj nie powinno być same <p> tylko musi to siedzieć w <div> lub najlepiej w <main>

Reply:
ok ale co